README.md

```diff
@@ -38,11 +62,94 @@ This has to be added using
 Pkg.clone("https://github.com/marcusps/ExpmV.jl.git")
 ```
 
-**Usage**
-
-QuDynamics.jl
-
+Usage
+-----
 The current package can be used by
 ```julia
 Pkg.clone("https://github.com/JuliaQuantum/QuDynamics.jl.git")
 ```
+
+We could start using the package by :
+```julia
+using QuDynamics
```

Comment:
I would add `using QuBase` since otherwise you don't get the functions to create states etc.